src/Donations/index.tsx

```diff
@@ -255,13 +255,15 @@ function DonationInfo() {
                   className="title-text text-white"
                   style={{ marginTop: "10px" }}
                 >
-                  {projectDetails.name}
+                  {paymentSetup.name && paymentSetup.name}
```

Comment:
@Shreyaschorge Change this to 

`{paymentSetup.name ? paymentSetup.name : ""}`

like you've done for description. This is in order to keep a consistent style.

Reply:
Okay